### PR TITLE
Add changes to ORM to get it working with postgres

### PR DESCRIPTION
**Pull request by charliecrouse**
I also added a ton of comments to make sure we are on the same page.  LMK

### CODE REVIEW

**Comment by charliecrouse** [comment]
> Looks good. What was the issue?
There was an issue with the ProjectMeta table.  filename field was conflicting with the filename fields of the commit linked or something. But I also added the appropriate relationship links to the project model

File: lamby/models/commit.py [hunk 1]
```diff
@@ -2,14 +2,45 @@
...
+
+    # PROJECT_ID -- (ForeignKey to Project)
+    project_id = db.Column(
+        db.Integer,
+        db.ForeignKey('project.id'),
+        nullable=False
+    )
+
+    # FILENAME -- Name of the model tied to this commit
+    filename = db.Column(db.String(120), unique=True, nullable=False)
```
Comment (charliecrouse) [inline lamby/models/commit.py]: This should be not unique i think

**Comment by charliecrouse** [comment]
This is the one

**Comment by pkoushik** [comment]
RIP CIRCLE CI

**Comment by pkoushik** [comment]
Coming back

**Pull request closed without merging**